File: 3753/3753.cs
public class Solution {
    public int TotalWavinessOfNumbersInRangeI(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(TotalWavinessOfNumbersInRangeI(root.left), TotalWavinessOfNumbersInRangeI(root.right));
    }
}

File: 3753/3753.go
func totalWavinessOfNumbersInRangeI(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := totalWavinessOfNumbersInRangeI(root.Left)
	r := totalWavinessOfNumbersInRangeI(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}